app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivityMode.java

```diff
@@ -0,0 +1,6 @@
+package fr.free.nrw.commons.nearby;
+
+enum NearbyActivityMode {
+    MAP,
+    LIST
+}
```

Comment:
If you run with the suggestions from NearbyActivity, this class would look quite different -
```java
enum NearbyActivityMode {
    MAP(R.drawable.ic_list_white_24dp),
    LIST(R.drawable.ic_map_white_24dp);

    @DrawableRes
    private final int icon;

    NearbyActivityMode(int icon) {
        this.icon = icon;
    }

    public NearbyActivityMode toggle() {
        return isMap() ? LIST : MAP;
    }

    public boolean isMap() {
        return MAP.equals(this);
    }

    public static NearbyActivityMode fromBoolean(boolean value) {
        return value ? MAP : LIST;
    }

    @DrawableRes
    public int getIcon() {
        return icon;
    }
}
```